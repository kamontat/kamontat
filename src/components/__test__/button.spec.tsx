import React from "react"
import tw from "twin.macro"
import renderer from "react-test-renderer"

const Button = tw.div`
  flex
`

test("Button renders correctly", () => {
  expect(renderer.create(<Button>This is hotpink.</Button>).toJSON()).toMatchSnapshot()
})
