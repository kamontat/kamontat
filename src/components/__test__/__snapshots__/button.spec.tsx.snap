// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

<div
  className="emotion-0 emotion-1"
>
  This is hotpink.
</div>
`;
